modules/nw-osp-enabling-ovs-offload.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-enabling-ovs-offload_{context}"]
+= Enabling OVS hardware offloading
+
+For clusters that run on {rh-openstack-first}, you can enable link:https://www.openvswitch.org/[Open vSwitch (OVS)] hardware offloading. 
+
+OVS is a multi-layer virtual switch that enables large-scale, multi-server network virtualization. 
+
+.Prerequisites
+
+* You installed a cluster on {rh-openstack} that is configured for single-root input/output virtualization (SR-IOV).
+* You installed the SR-IOV network Operator on your cluster. 
+* You created two `hw-offload` type virtual function (VF) interfaces on your cluster.
+
+.Procedure
+
+. From a command line, disable the admission webhook:
++
+[source,terminal]
+----
+$ oc patch sriovoperatorconfig default --type=merge -n openshift-sriov-network-operator --patch '{ "spec": { "enableOperatorWebhook": false } }'
+----
+
+. Create an `SriovNetworkNodePolicy` policy for the two `hw-offload` type VF interfaces that are on your cluster:
++
+.The first virtual function interface
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: "hwoffload9"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true
+  nicSelector:
+    pfNames:
+    - ens6
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: 'true'
+  numVfs: 1
+  priority: 99
+  resourceName: "hwoffload9"
+----
++
+.The second virtual function interface
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy <1>
+metadata:
+  name: "hwoffload10"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true
+  nicSelector:
+    pfNames: <2>
+    - ens5
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: 'true'
+  numVfs: 1
+  priority: 99
+  resourceName: "hwoffload10"
+----
+<1> Insert the `SriovNetworkNodePolicy` value here.
+<2> Both interfaces must include physical function (PF) names.
```

Comment:
Do these only apply to the second virtual function interface? If they apply to the first one also, I think that file could use the annotations too.